x/gov/client/rest/rest.go

```diff
@@ -476,54 +478,35 @@ func queryProposalsWithParameterFn(cdc *wire.Codec) http.HandlerFunc {
 				return
 			}
 		}
+		if len(strNumLatest) != 0 {
+			numLatest, err = strconv.ParseInt(strNumLatest, 10, 64)
```

Comment:
Would be good to create a helper function. I am sure this is not the only case where we parse int64. No?

```
func parseInt64OrReturnBadRequest(s string, w http. ResponseWriter) (n int64, ok bool) {
numLatest, err = strconv.ParseInt(s, 10, 64)
if err != nil {
				w.WriteHeader(http.StatusBadRequest)
				err := errors.Errorf("'%s' is not a valid int64", s)
				w.Write([]byte(err.Error()))
				return
			}
}
```